### DISCUSSION TITLE
MQTT publish, subscribe

### DISCUSSION THREAD

**Discussion by hana-gharrad** [gama-platform/gama#48] [Q&A] [answered]
Hello again,

I was trying some examples with MQTT messaging. I tried some send and receive and it work well.
Now, I want to use the concept of create and publish topics and subscribe to topic. I did not find something similar in the tutorials or the documentation.
Any guidelines please? 

Many thanks,

**Comment by benoitgaudou** [comment, marked as answer]
Dear,
I do not know whether this question is still relevant ...
The subscribe of MQTT is not directly available in GAMA, but topic is in
fact agents name, and you can have group of agents (
https://gama-platform.org/wiki/next/BuiltInSkills#join_group) has a more
general topic.
If you need more feature, do not hesitate to add an enhancement request.

Cheers

Benoit

Le mar. 4 juil. 2023 à 13:12, hana-gharrad ***@***.***> a
écrit :

**Reply by hana-gharrad** [reply]
Dear Benoit,

Thank you for the reply. I did use the name of the agent and it is working fine and serve the purpose I used it for.

Thanks,
